src/utils/todo/action-functions.ts

```diff
@@ -128,7 +128,7 @@ export const doResumeFiling = (item: TodoItemI): void => {
       // navigate to Create UI to resume this Dissolution
       navigateFn = goToCreatePage
       path = '/define-dissolution'
-      params = { id: bootstrapIdentifier }
+      params = { id: currentBusinessIdentifier }
```

Comment:
Does this work for bootstrap filings?

Reply:
Dissolution should be an option for bootstrap businesses.

Reply:
No, you cannot dissolve a bootstrap business, because it doesn't exist. It's actually just a filing with a "Txxxxxxx" number assigned to it that we make look like a business.